common/src/main/java/io/pravega/common/util/CollectionHelpers.java

```diff
@@ -66,4 +72,187 @@
     public static <T> Collection<T> filterOut(Collection<T> collection, Collection<T> toExclude) {
         return collection.stream().filter(o -> !toExclude.contains(o)).collect(Collectors.toList());
     }
+
```

Comment:
Add tests for these

Reply:
I did. Check out the changes to `CollectionHelpersTests`.